Gopkg.toml

```diff
@@ -64,23 +64,23 @@ ignored = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.12.0"
+  version = "kubernetes-1.13.0-alpha.1"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.12.0"
+  version = "kubernetes-1.13.0-alpha.1"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.12.0"
+  version = "kubernetes-1.13.0-alpha.1"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.12.0"
+  version = "v9.0.0"
```

Comment:
need kubernetes-1.13.0-alpha.1  as well, there is something different between 1.13 and v9.0.0 in client-go/transport

Reply:
Right. my bad, fixing